roles/swift-account/tasks/main.yml

```diff
@@ -11,7 +11,6 @@
 - name: account-server config
   template: src=etc/swift/account-server.conf mode=0640
             dest=/etc/swift/account-server.conf owner=swift group=swift
-  notify: restart swift-account-services
 
```

Comment:
Removing restart checks since they would be handled by upgrade path if needed... For fresh deploy start tasks will be used